.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -1,27 +1,61 @@
 ---
-name: Create new doc type template
-about: For issues where a templateer needs to create a new doc type template.
-title: Create {doc_type} template
+name: Create new content type template
+about: For issues where a templateer needs to create a new content type template.
+title: Create {content_type} template
 labels: templates
 assignees: ''
 
 ---
+# Title
+{Keep the title short, yet descriptive.
+Here are some examples:
+* Create a new {content_type} template 
+* Align the existing {content_type} template to new contributing guidelines
+}
 
-# Create {doctype} template
+## Before you begin
+{Optional: In this section specify any pre-requisites, background knowledge that the templateer should know before working on a template.}
+
+1. Join The Good Docs Project [slack workspace](https://thegooddocs.slack.com/). 
+2. Read the [template contributing process](https://github.com/thegooddocsproject/templates/blob/dev/CONTRIBUTING.md#overview-of-the-template-writing-phases).
+3. You are strongly encouraged to join one of the [working groups](https://thegooddocsproject.dev/working-group/) to get valuable support from the community such as mentorship, Git training, and helpful feedback as you contribute to your first template.
+4. Request access to the `templates` repository by joining the **#templates** channel in slack and posting a request. 
+5. Assign yourself to an issue for the template that you want to work on.
+6. Add the issue to the [templates-kanban](https://github.com/thegooddocsproject/templates/projects/1) and move it into the `Research-phase` column.
+7. While it is not mandatory to know Git to start contributing, if you are keen, you can familiarize yourself with basic git commands and understand the workflow for contributing on GitHub.  
 
 ## Description
-### Doc Type Name: {doc_type}
+{Provide specific details about the template. You could adopt a user story approach to describe a new {content_type} template.  
+
+_As a [type of user] I want [my goal] so that [my reason]._  
+
+Example:
+_As a developer, I want an installation template to document the install instructions for my application._ 
+
+Keep the following points in mind while writing the description:
+* If applicable, always include the relevant people using `@{Username}` who can support in resolving the issue. It can be someone who has previously worked on the template. Also, indicate the relevant slack channel where the templateer can reach out to the larger community. 
+* Use bullet points and subheadings to structure complex details instead of big paragraphs.
+* Add links to any document references.
+}
+
+### Doc Type Name: {content_type}
```

Comment:
Question (non-blocking): Should this one also be "Content type name" for consistency?

Also FYI, we follow the Google Developer's Style Guide so the heading should be in sentence case rather than title case. :)

Reply:
Fixed